packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -669,8 +683,9 @@ const PipelineEditor = forwardRef(
                     label: "Node Properties",
                     icon: theme.overrides?.propertiesIcon,
                     content: (
-                      <PropertiesPanel
+                      <NodeProperties
                         selectedNodes={selectedNodes}
+                        pipeline={pipeline}
```

Comment:
we could probably pull the pipeline logic out of here and just do the mapping above and pass the mapped nodes to the selectedNodes prop